frontend/src/components/postition-templates-list.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export class PositionTemplatesList extends React.Component {
+    static propTypes = {
+        position_templates: PropTypes.arrayOf(
+            PropTypes.shape({
+                position_type: PropTypes.string,
+                offer_template: PropTypes.string
+            })
+        ).isRequired
+    };
+    render() {
+        const positionTypeStyle = {
+            color: "blue"
+        };
+        const { position_templates } = this.props;
+        let templateList = <div>No Templates...</div>;
+        if (position_templates.length > 0) {
+            templateList = (
+                <ul>
+                    {position_templates.map(template => (
+                        <li key={template.offer_template}>
+                            <span style={positionTypeStyle}>
```

Comment:
You should almost never use inline styles... (basically you only use them when drafting components and then you remove them later). Use an actual `className` and put some css into a corresponding css file.